x/rollup/keeper/msg_server_test.go

```diff
@@ -0,0 +1,157 @@
+package keeper_test
+
+import (
+	"encoding/json"
+
+	"cosmossdk.io/math"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/ethereum-optimism/optimism/op-node/rollup/derive"
+	"github.com/ethereum-optimism/optimism/op-service/eth"
+	"github.com/golang/mock/gomock"
+	"github.com/polymerdao/monomer/testutils"
+	"github.com/polymerdao/monomer/x/rollup/keeper"
+	"github.com/polymerdao/monomer/x/rollup/types"
+)
+
+func (s *KeeperTestSuite) TestApplyL1Txs() {
```

Comment:
Is the SDK convention to use a suite? We don't use suites elsewhere in the codebase. Any thoughts on the advantages and disadvantages?

Reply:
Noting for visibility - we talked during our daily call and addressed this question. Test suites are nice for cosmos sdk modules since you can store all relevant keeper/store info on them and reset the stores/keepers before every subtest automatically